tests/pytorch/nightly/llama2-model.libsonnet

```diff
@@ -114,12 +114,6 @@ local utils = import 'templates/utils.libsonnet';
         # install tokenizer model
         wget https://storage.googleapis.com/tpu-pytorch/lsiyuan-experiment/llama/spiece.model
 
-        # git clone and build llama
-        git clone --branch llama2-google-next-inference https://github.com/pytorch-tpu/llama.git
```

Comment:
I understand we don't need this cloned for training. Where do you clone llama2 for inference?

Reply:
we clone llama2 for inference in pjrt setup: https://github.com/ManfeiBai/ml-testing-accelerators/blob/96a8320b941d0bf9023657bd2e18ea8fedd03247/tests/pytorch/nightly/llama2-model.libsonnet#L69